src/MiniProfiler.Shared/ui/includes.js

```diff
@@ -216,7 +216,7 @@ var MiniProfiler = (function () {
                     customTiming.ParentTimingId = timing.Id;
                     customStat.Duration += customTiming.DurationMilliseconds;
                     customStat.Count++;
-                    if (customTiming.CommandString && duplicates[customTiming.CommandString]) {
+                    if (customTiming.CommandString && duplicates[customTiming.CommandString] && !ignoreDuplicateCustomTiming(customTiming)) {
```

Comment:
I'd change this to be simpler: remove this change and instead modify the `else` below to `else if (!ignoreDuplicateCustomTiming(customTiming))`, so it never gets added to `duplicates` below, that is less likely to get tripped on for uses of `duplicates` later.